src/Controller/ProfileController.php

```diff
@@ -112,7 +113,7 @@ public function packagesAction(Request $req, #[VarName('name')] User $user, Favo
     }
 
     #[Route(path: '/profile/edit', name: 'edit_profile')]
-    public function editAction(Request $request): Response
+    public function editAction(Request $request, UserNotifier $userNotifier): Response
     {
         $user = $this->getUser();
         if (!is_object($user)) {
```

Comment:
Just an idea, doing this here instead would remove the need for the check below.. it keeps things a little cleaner in the main logic of the function

```suggestion
        if (!$user instanceof User) {
```